src/data/lovelace.ts

```diff
@@ -44,6 +44,7 @@ export interface NavigateActionConfig {
 
 export interface MoreInfoActionConfig {
   action: "more-info";
+  entity?: string;
```

Comment:
I don't think that this should be optional? How do we know whose more info to open ?

Reply:
good point.

Reply:
It's just annoying because there is not always an entity in the context, like with some of the picture elements.

Reply:
We should probably make it possible to pass in what schema to use

Reply:
if there is not an entity in the context, we just won't attempt to show more-info. In that case, should we perhaps create a toast message stating that it was not defined?

Reply:
I rather have our config editor not allow our users to mess up than handle it gracefully when our users mess up.

Reply:
could perhaps pass in config.entity where applicable to the actioneditor as the default when more-info is selected? we can then make it required and in most cases a user won't have to do extra steps

Reply:
That would make this a breaking change...

Reply:
Correct solution:

We have to introduce a new "fallback-entity" to the action editor. Actions that rely on entities will allow entity to be optional. Written config won't store the entity, so that we always take the one from context.

Quick solution: we provide a default entity and that is written when editing an action that requires an entity but it has none.